src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritQueryHandler.java

```diff
@@ -43,16 +43,16 @@
  */
 public class GerritQueryHandler {
 
-    private static final Logger logger = LoggerFactory.getLogger(GerritQueryHandler.class);
+    static final Logger logger = LoggerFactory.getLogger(GerritQueryHandler.class);
     /**
      * The base of the query ssh command to send to Gerrit.
      */
     public static final String QUERY_COMMAND = "gerrit query";
-    private final String gerritHostName;
-    private final int gerritSshPort;
-    private final String gerritProxy;
-    private final Authentication authentication;
-    private final int connectionTimeout;
+    final String gerritHostName;
```

Comment:
Why?
I might be blind but I don't see any reason in the code to change the access level of these fields?

Reply:
You are right. I thought they are referenced in the subclass but they are not (anymore).
I changed it back, except for logger, that one is needed in the subclass.

Reply:
If it is only needed in the sub classes it should be `protected` instead of package level protection.

Reply:
Makes sense. Done.